modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/Deployment.java

```diff
@@ -17,7 +17,7 @@ public interface Deployment {
 
     DeployResult deploy(DeployTask task);
 
-    DeployResult destroy(DeployTask task);
+    DeployResult destroy(DeployTask task, String tfState);
```

Comment:
Shoud not use tfState here, for tfState is just for terraformDeployment. should not be exposed in the skeleton.